.github/workflows/create-tag.yml

```diff
@@ -0,0 +1,80 @@
+name: create-tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      TAG:
+        description: 'TAG'
+        required: true
+        default: ""
+        type: string
+      TAG_BODY:
+        description: 'TAG DESCRIPTION'
+        required: false
+        default: ""
+        type: string
+      IS_DRAFT:
+        description: 'CREATE AS DRAFT'
+        required: false
+        default: false
+        type: boolean
+      IS_PRERELEASE:
+        description: 'PRERELEASE TAG'
+        required: true
+        default: true
+        type: boolean
+
+permissions:
+  contents: write
+
+jobs:
+  create-tag:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set Node.js 20.x
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Rebuild the dist/ directory
+        run: |
+          export NODE_OPTIONS=--openssl-legacy-provider
+          npm run build && npm run package
+
+      - name: Compare the expected and actual dist/ directories
+        run: |
+          if [ "$(git diff --ignore-space-at-eol dist/ | wc -l)" -gt "0" ]; then
+            echo "Detected uncommitted changes after build.  See status below:"
+            git diff
+            exit 1
+          fi
+        id: diff
+
+      - name: Process tag body
+        id: process_body
+        run: |
+          tag_body="${{ github.event.inputs.TAG_BODY }}"
+          echo "${tag_body}">>processed_body.txt
+          sed -i 's/\\n/\n/g' processed_body.txt
+          cat processed_body.txt
+          processed_body=$(cat processed_body.txt)
+          echo "PROCESSED_BODY<<EOF">>$GITHUB_ENV
+          echo "$processed_body">>$GITHUB_ENV
+          echo "EOF">>$GITHUB_ENV
+
+      - name: Create tag
+        id: create_tag
+        uses: actions/create-release@latest
```

Comment:
actions/create-release@latest is deprecated
The set-output command is deprecated (though not directly used here)

Should we use different actions to release?